src/modules/gnd_att_control/GroundRoverAttitudeControl.hpp

```diff
@@ -77,14 +78,15 @@ class GroundRoverAttitudeControl
 
 	bool		_task_should_exit{false};		/**< if true, attitude control task should exit */
 	bool		_task_running{false};			/**< if true, task is running in its mainloop */
-	int		_control_task{-1};			/**< task handle */
+	int		_control_task{ -1};			/**< task handle */
 
-	int		_att_sp_sub{-1};			/**< vehicle attitude setpoint */
-	int		_battery_status_sub{-1};		/**< battery status subscription */
-	int		_ctrl_state_sub{-1};		/**< control state subscription */
-	int		_manual_sub{-1};			/**< notification of manual control updates */
-	int		_params_sub{-1};			/**< notification of parameter updates */
-	int		_vcontrol_mode_sub{-1};		/**< vehicle status subscription */
+	int		_att_sp_sub{ -1};			/**< vehicle attitude setpoint */
+	int		_battery_status_sub{ -1};		/**< battery status subscription */
+	int		_ctrl_state_sub{ -1};		/**< control state subscription */
+	int		_manual_sub{ -1};			/**< notification of manual control updates */
+	int		_params_sub{ -1};			/**< notification of parameter updates */
```

Comment:
Did you run this through astyle? If so what version are you using, this seems wrong?

Reply:
Yes, but I used 2.05.1. I will update my astyle version. 

It should be less annoying now. Thanks!